runtime/test/planner-tests.js

```diff
@@ -194,7 +186,7 @@ describe('ConvertConstraintsToConnections', async() => {
     let { result, score } = results[0];
     assert.deepEqual(result.toString(),
 `recipe
-  map as view0
+  use as view0
```

Comment:
The ConvertConnectionsToConstraint strategy doesn't currently work with map. Since it generates connections that use `inout`.